### ISSUE TITLE
Unify parameter notation for all scalers

### ISSUE THREAD

**Issue by tomkerkhove** [kedacore/keda-docs#540]
### Discussed in https://github.com/kedacore/keda-docs/discussions/539

<div type='discussions-op-text'>

<sup>Originally posted by **JorTurFer** September 22, 2021</sup>
We don't have defined the notation that should be use when you are explaining things like parameters in docs. That's why you can find notations like these:
- `Parameter` - Description
- `Parameter`: Description
- `Parameter:` Description

In the same way, in some cases the sentences finish with a dot and in other cases don't. 

I'd like to review all the docs and unify the styles, but maybe the styles should be decided before it.

WDTY?</div>

**Comment by tomkerkhove** [comment]
All scalers should follow this convention:
> `name` - Description.

@JorTurFer is willing to pick this up, thank you!

**Comment by JorTurFer** [comment]
The notation will be (just to remind it to myself):
`name` - Description.

**Comment by JorTurFer** [comment]
One important thing @tomkerkhove ,
Should I update only v2.5 docs or all? I'm asking because IDK if changes in the old files are being deployed, I suppose that yes but just to double check it

**Comment by tomkerkhove** [comment]
Yes, please update all versions

**Comment by JorTurFer** [comment]
Hi @tomkerkhove ,
I'm working on this and I found that we are saying if the parameter is optional in different ways like:
- `name` - Optional. Description.
- `name` - Description. Optional.
- `name` - Description. *(Optional)*
- More ways those I haven't discovered yet

WDTY if I unify it also with, for example this?
- `name` - Description. Optional.

**Comment by tomkerkhove** [comment]
Let's go with this:
> - `name` - Description. *(Optional)*

**Comment by tomkerkhove** [comment]
We should also document this convention in https://github.com/kedacore/keda-docs/blob/master/CONTRIBUTING.md (preferably a separate PR)

**Comment by JorTurFer** [comment]
I'll do it when the "main PR" is ready to merge. I'll document how the parameters should be notated (I'd like to wait until we have a "final version there" because we are having to do some small changes in the pattern to match all cases)
